packages/api/src/command/medical/document/check-doc-queries.ts

```diff
@@ -91,27 +106,29 @@ export async function getPatientsToUpdate(
 }
 
 function getQuery(
-  propertyName: "convert" | "download",
+  propertyName: keyof Pick<DocumentQueryProgress, "convert" | "download">,
   maxTime: duration.Duration,
   patientIds: string[] = []
 ): string {
-  // START - Literally just making sure we bring our attention to this query if we change something with the patient model
-  // so we update the query as well
-  const voidPatient = { data: {} } as PatientModel;
-  const c: Progress = voidPatient.data.documentQueryProgress?.convert ?? ({} as Progress);
-  [c?.status, c?.total, c?.successful, c?.errors];
-  const d: Progress = voidPatient.data.documentQueryProgress?.download ?? ({} as Progress);
-  [d?.status, d?.total, d?.successful, d?.errors];
+  // START - Workaround to force a compilation error if we change something on the model, since we're
+  // using raw SQL here.
+  const data: keyof Pick<PatientCreate, "data"> = "data";
+  const documentQueryProgress: keyof Pick<PatientData, "documentQueryProgress"> =
+    "documentQueryProgress";
+  const status: keyof Pick<Progress, "status"> = "status";
+  const total: keyof Pick<Progress, "total"> = "total";
+  const successful: keyof Pick<Progress, "successful"> = "successful";
+  const errors: keyof Pick<Progress, "errors"> = "errors";
```

Comment:
had already built this when I realized we had another check... ended up keeping this b/c it seems more integrated w/ the query, so I think we have higher chances to keep this pattern on the native query below if we need to add new columns, for example.